apps/meteor/client/views/admin/rooms/EditRoom.tsx

```diff
@@ -97,7 +97,7 @@ const EditRoom = ({ room, onChange, onDelete }: EditRoomProps) => {
 	const handleArchive = useArchiveRoom(room);
 
 	const handleUpdateRoomData = useEffectEvent(async ({ isDefault, roomName, favorite, ...formData }) => {
-		const data = getDirtyFields(formData, dirtyFields);
```

Comment:
since roomName is not passed due to which edit_room_name functionality was not working